playpen/mongoengine/dockerimage/rhel-pulp.repo

```diff
@@ -4,6 +4,15 @@
 [pulp-v2-stable]
 name=Pulp 2.8 Stable Builds
 baseurl=http://repos.fedorapeople.org/repos/pulp/pulp/stable/2.8/7Server/$basearch/
+enabled=0
+skip_if_unavailable=0
+gpgcheck=1
+gpgkey=https://repos.fedorapeople.org/repos/pulp/pulp/GPG-RPM-KEY-pulp-2
+
+# Version 2.8 Beta Builds
+[pulp-v2-beta]
+name=Pulp 2.8 Beta Builds
+baseurl=http://repos.fedorapeople.org/repos/pulp/pulp/beta/2.8/7Server/$basearch/
```

Comment:
With an `ADD` statement it sounds like the `run.sh` is bundled with the docker image which is good.

The `run.sh` is merged forward to master already so wouldn't a rebuild of the docker image include the up to date `run.sh` even though the version in `2.8.3-0.1.beta` is not up to date?

If ^ is true, then all we need to do is have docker rebuild the image after the `run.sh` was merged forward to master. Perhaps we should keep the enabled repo used by the Dockerfile to be the GA release, what do you think?